### PR TITLE
Remove disclaimer

### PR DESCRIPTION
**Pull request by shreya-mishra**
Removed disclaimer in which bot mentions everyone and says  Disclaimer: Very soon all the users will be tagged individually in a separate(new) message!

### CODE REVIEW

File: src/utils/checkDisplayType.ts [hunk 1]
```diff
@@ -2,16 +2,13 @@
...
   usersWithMatchingRole,
 }: {
   msgToBeSent?: string;
   usersWithMatchingRole: string[];
 }) {
   if (usersWithMatchingRole.length > 0) {
-    const returnString = msgToBeSent ? msgToBeSent : "";
-
-    return `${returnString} ${usersWithMatchingRole} \n \`Disclaimer: Very soon all the users will be tagged individually in a separate(new) message!\``;
+    return `${usersWithMatchingRole}`;
```
Comment (RitikJaiswal75) [inline src/utils/checkDisplayType.ts:11]: why did we remove the returnString? mention each also includes the message that the users send, if we just return the users with matching roles then we would never get the message.

File: tests/unit/handlers/mentionEachUser.test.ts
Comment (RitikJaiswal75) [inline tests/unit/handlers/mentionEachUser.test.ts:1]: please test for sending custom message as well

**Comment by sahsisunny** [comment]
Please mention the issue ticket.

**Review by sahsisunny — APPROVED** [review]

**Review by Pratiyushkumar — APPROVED** [review]

**Review by iamitprakash — APPROVED** [review]

File: tests/unit/handlers/mentionEachUser.test.ts
Comment (shreya-mishra) [reply tests/unit/handlers/mentionEachUser.test.ts:1]: Test will be moving to next PR as Ankush said

File: src/utils/checkDisplayType.ts (hunk 1 above)
Comment (shreya-mishra) [reply src/utils/checkDisplayType.ts]: resolved it in last commit

**Pull request merged**